src/System Application/App/Azure Blob Services API/src/Helper/ABSOptionalParameters.Codeunit.al

```diff
@@ -403,6 +403,16 @@ codeunit 9047 "ABS Optional Parameters"
         SetParameter('include', "Value");
     end;
 
+    /// <summary>
+    /// Specifies the datasets to be returned in the response.
```

Comment:
Doc is off,

Reply:
i don't understand your comment. can you please repeat this in more detail?

Reply:
Your code documentation summary states "Specifies the datasets to be returned in the response.", just as the parameter description says "The dataset to include in text". That's not what the ShowOnly function does. Looks like a copy/paste from above.

Reply:
Confirm. No copy from above. Copy/Paste from msft learn: https://learn.microsoft.com/en-us/rest/api/storageservices/list-blobs?tabs=microsoft-entra-id#uri-parameters
I did copy the listing of the possible values since we probably won't notice immediately if something changes there.
every developer who works with the azure blob looks there anyway.

Reply:
My bad then.